manager/src/main/java/org/openremote/manager/rules/flow/NodeExecutionRequestInfo.java

```diff
@@ -205,6 +228,8 @@ public RulesFacts getFacts() {
         return facts;
     }
 
+    public Logger getLog(){return LOG;}
```

Comment:
We like spaces and new lines in our code base, but anyway this can be removed just make the `LOG` field `protected`.
```
    public Logger getLog() {
    return LOG;}
```